packages/themes/scss/_mixins.scss

```diff
@@ -1,170 +1,35 @@
-  $text-03: map-get($theme, 'text-03') !global;
-  $text-04: map-get($theme, 'text-04') !global;
-  $text-05: map-get($theme, 'text-05') !global;
-  $icon-01: map-get($theme, 'icon-01') !global;
-  $icon-02: map-get($theme, 'icon-02') !global;
-  $icon-03: map-get($theme, 'icon-03') !global;
-  $link-01: map-get($theme, 'link-01') !global;
-  $inverse-link: map-get($theme, 'inverse-link') !global;
-  $field-01: map-get($theme, 'field-01') !global;
-  $field-02: map-get($theme, 'field-02') !global;
-  $inverse-01: map-get($theme, 'inverse-01') !global;
-  $inverse-02: map-get($theme, 'inverse-02') !global;
-  $support-01: map-get($theme, 'support-01') !global;
-  $support-02: map-get($theme, 'support-02') !global;
-  $support-03: map-get($theme, 'support-03') !global;
-  $support-04: map-get($theme, 'support-04') !global;
-  $inverse-support-01: map-get($theme, 'inverse-support-01') !global;
-  $inverse-support-02: map-get($theme, 'inverse-support-02') !global;
-  $inverse-support-03: map-get($theme, 'inverse-support-03') !global;
-  $inverse-support-04: map-get($theme, 'inverse-support-04') !global;
-  $overlay-01: map-get($theme, 'overlay-01') !global;
-  $danger: map-get($theme, 'danger') !global;
-  $focus: map-get($theme, 'focus') !global;
-  $inverse-focus-ui: map-get($theme, 'inverse-focus-ui') !global;
-  $hover-primary: map-get($theme, 'hover-primary') !global;
-  $active-primary: map-get($theme, 'active-primary') !global;
-  $hover-primary-text: map-get($theme, 'hover-primary-text') !global;
-  $hover-secondary: map-get($theme, 'hover-secondary') !global;
-  $active-secondary: map-get($theme, 'active-secondary') !global;
-  $hover-tertiary: map-get($theme, 'hover-tertiary') !global;
-  $active-tertiary: map-get($theme, 'active-tertiary') !global;
-  $hover-ui: map-get($theme, 'hover-ui') !global;
-  $active-ui: map-get($theme, 'active-ui') !global;
-  $selected-ui: map-get($theme, 'selected-ui') !global;
-  $hover-selected-ui: map-get($theme, 'hover-selected-ui') !global;
-  $inverse-hover-ui: map-get($theme, 'inverse-hover-ui') !global;
-  $hover-danger: map-get($theme, 'hover-danger') !global;
-  $active-danger: map-get($theme, 'active-danger') !global;
-  $hover-row: map-get($theme, 'hover-row') !global;
-  $visited-link: map-get($theme, 'visited-link') !global;
-  $disabled-01: map-get($theme, 'disabled-01') !global;
-  $disabled-02: map-get($theme, 'disabled-02') !global;
-  $disabled-03: map-get($theme, 'disabled-03') !global;
-  $highlight: map-get($theme, 'highlight') !global;
-  $skeleton-01: map-get($theme, 'skeleton-01') !global;
-  $skeleton-02: map-get($theme, 'skeleton-02') !global;
-  $brand-01: map-get($theme, 'brand-01') !global;
-  $brand-02: map-get($theme, 'brand-02') !global;
-  $brand-03: map-get($theme, 'brand-03') !global;
-  $active-01: map-get($theme, 'active-01') !global;
-  $hover-field: map-get($theme, 'hover-field') !global;
+$custom-property-prefix: 'cds' !default;
```

Comment:
Can this go in a `_variables.scss` file?

Reply:
I wanted to keep this internal-only for now versus letting folks mess with it, and try and scope it with the mixin.

Separately, this is one reason I'm not a big fan of file naming per syntax instead of feature because functionality wise it'd be great to have these in the same file, otherwise it's hard to know where things are coming from.

Reply:
Ah, it's designated as private. I don't have a preference on where private variables live. Feel free to resolve this convo.